src-test/org/etools/j1939_84/controllers/part01/Part01Step04ControllerTest.java

```diff
@@ -284,7 +399,7 @@ public void testGetStepNumber() {
         assertEquals(STEP_NUMBER, instance.getStepNumber());
     }
 
-    @Test
+//    @Test
```

Comment:
Instead of commented out the tag, you should put `@Ignore` on it so we know the test was ignored in the reports.